### PR TITLE
Added support for Pi2Media CSB503SPD2 audio card

### PR DESCRIPTION
**Pull request by Pi2Design**

### CODE REVIEW

**Comment by Pi2Design** [comment]
Unlike the pull request for the CSB503HTA, this is not a simple clone. We've added support for GPIO controlled oscillator source on the wm8804. The hardware works well and is tested.

**Comment by Pi2Design** [comment]
We are looking for feedback on this.

**Comment by pelwell** [comment]
Good timing - it was open in my browser when your message arrived.

File: arch/arm/boot/dts/overlays/README [hunk 1]
```diff
@@ -673,6 +673,10 @@ Params: speed                   Display SPI bus speed
 
 [ The pcf8563-rtc overlay has been deleted. See i2c-rtc. ]
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/README:675]: Two blank lines between entries, please.

File: arch/arm/boot/dts/overlays/README [hunk 2]
```diff
@@ -673,6 +673,10 @@ Params: speed                   Display SPI bus speed
...
 
+Name:   pi2media-csb503spd2-audio
+Info:   Configures the Pi2Media CSB503SPD2 audio card
+Load:   dtoverlay=pi2media-csb503spd2-audio
+Params: <None>
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/README:680]: And here.

**Comment by pelwell** [comment]
Apart from the README formatting, the problem with this PR is the same as with #1583 - you should be using Device Tree for the GPIOs, and devm_gpiod_get is the preferred API function to locate them.

**Comment by Ruffio** [comment]
@Pi2Design is this PR going to be updated according to @pelwell descriptions/requirements ? If not, then I don't think this PR will ever be accepted.

**Comment by Ruffio** [comment]
@Pi2Design is this PR going to be updated according to @pelwell descriptions/requirements ? If not, then I don't think this PR will ever be accepted.

**Comment by pelwell** [comment]
Closing for lack of activity.

**Pull request closed without merging**